blog/2017-09-25-streaming-to-another-database.adoc

```diff
@@ -0,0 +1,222 @@
+If a downstream connector is able to handle those events it is possible to configure `UnwrapFromEnvelope` to pass them downstream.
+
+=== Topic naming
+Last but not least there is a difference in naming of topics.
+Debezium uses fully qualified naming for target topics representing each table it manages.
+The naming follows the pattern `<instance-name>,<database-name>.<table-name>`.
+Kafka Connect JDBC Connector works with simple names `<table-name>`.
+
+In complex scenarios the user would deploy https://kafka.apache.org/documentation/streams/[Kafka Streams] framework to establish elaborated routing between source and target routes.
+In our example we will use a stock `RegexRouter` SMT that would route records created by Debezium into topics named according to JDBC Connector schema.
+[source,indent=0]
+----
+        "transforms": "route",
+        "transforms.route.type": "org.apache.kafka.connect.transforms.RegexRouter",
+        "transforms.route.regex": "([^.]+)\\.([^.]+)\\.([^.]+)",
+        "transforms.route.replacement": "$3"
+----
+
+== Example
+Kick the tires and let's try our example!
+
+First of all we need to deploy all components.
+[source,bash,indent=0]
+----
+export DEBEZIUM_VERSION=0.6
+docker-compose up
+----
+
+When all components are started we are going to register JDBC Sink connector writing into PostgreSQL database
+[source,bash,indent=0]
+----
+curl -i -X POST -H "Accept:application/json" -H  "Content-Type:application/json" http://localhost:8083/connectors/ -d @jdbc-sink.json
+----
+
+using this registration request.
+[source,json,indent=0]
+----
+{
+    "name": "jdbc-sink",
+    "config": {
+        "connector.class": "io.confluent.connect.jdbc.JdbcSinkConnector",
+        "tasks.max": "1",
+        "topics": "customers",
+        "connection.url": "jdbc:postgresql://postgres:5432/inventory?user=postgresuser&password=postgrespw",
+        "transforms": "unwrap",                                                  (1)
+        "transforms.unwrap.type": "io.debezium.transforms.UnwrapFromEnvelope",   (1)
+        "auto.create": "true",                                                   (2)
+        "insert.mode": "upsert",                                                 (3)
+        "pk.fields": "id",                                                       (4)
+        "pk.mode": "record_value"                                                (4)
+    }
+}
+----
+
+The request configures this options
+
+1. unwrapping Debezium complex format into a simple one
+2. automatically create target tables
+3. insert a row if it does not exist or update an existing one
+4. identify primary key stored in Kafka's record value field
+
+[source,bash,indent=0]
+----
+curl -i -X POST -H "Accept:application/json" -H  "Content-Type:application/json" http://localhost:8083/connectors/ -d @source.json
+----
+The registration request contains SMT for routing records into topics with simple name.
```

Comment:
How about showing the source connector configuration like you did for the sink connector, and explaining some of the configuration parameters?